crates/core/component/ibc/src/lib.rs

```diff
@@ -18,7 +18,7 @@ pub mod params;
 mod version;
 
 mod prefix;
-pub use prefix::{IBC_COMMITMENT_PREFIX, IBC_PROOF_SPECS, IBC_SUBSTORE_PREFIX};
+pub use prefix::{MerklePrefixExt, IBC_COMMITMENT_PREFIX, IBC_PROOF_SPECS, IBC_SUBSTORE_PREFIX};
```

Comment:
The addition of exporting `MarklePrefixExt` via the penumbra-ibc crate is the only non-test change in the PR, and looks fine.